migration/v0.29/upgrade.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
As you have done such a nice script for upgrade, i would say yes you could move it to bin/apps